common/gcloud.py

```diff
@@ -86,7 +87,13 @@ def create_instance(instance_name: str,
         command.extend(
             ['--metadata-from-file', 'startup-script=' + startup_script])
 
-    return new_process.execute(command, expect_zero=False, **kwargs)[0] == 0
+    result = new_process.execute(command, expect_zero=False, **kwargs)
+    if result.retcode == 0:
+        return True
+
+    logs.info('Failed to create instance. Command: %s failed. Output: %s',
```

Comment:
logs.error ?

Reply:
It's not worth reporting as an error when we are creating runners and there is no quota. So I leave it up to the caller.